lightning/src/ln/channel.rs

```diff
@@ -7126,6 +7173,8 @@ impl<'a, 'b, 'c, ES: Deref, SP: Deref> ReadableArgs<(&'a ES, &'b SP, u32, &'c Ch
 			(29, temporary_channel_id, option),
 			(31, channel_pending_event_emitted, option),
 			(33, pending_monitor_updates, vec_type),
+			(35, pending_outbound_skimmed_fees_opt, optional_vec),
```

Comment:
I was thinking it only needed to be even on the receiving side, since on the forwarding side, ignoring this field would just result in a failed forward. Can change it though